index.js

```diff
@@ -54,12 +53,66 @@ class OktaStatusError extends Error {
   }
 }
 
+class ConfigurationValidationError extends Error {}
+const findDomainURL = 'https://bit.ly/finding-okta-domain';
```

Comment:
```suggestion
const findDomainURL = 'https://developer.okta.com/docs/guides/find-your-domain/';
```

We should use the actual full URL instead of a `bit.ly` link, since the target for that link could be changed in the future. I don't want to trust external resources out of our control.